contents/docs/data-warehouse/views.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Creating views
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+The PostHog data warehouse supports views. With views, you can save your most used queries and reference them in subsequent queries.
+
+### Creating a view
+
+Query views are created directly inside the HogQL editor. If the query has valid view characteristics, the "Save as view" button will be enabled. You will be prompted to give the view a name which will be the name that can be refeneced in other queries. For a query to be a valid view, all fields being accessed must be aliased. The alias names will be how you access the fields on the view. 
```

Comment:
```suggestion
Query views are created directly inside SQL insights. If the query has valid view characteristics, the "Save as view" button is enabled. When clicked, you are prompted to give the view a name which can then be referenced in other queries. 

For a query to be a valid view, all fields being accessed must be aliased. The alias names are how you access the fields on the view. 
```
what does "all fields being accessed must be aliased" mean, how does someone figure that out?

Reply:
The create a view button won't be clickable and will give you a tooltip that says to alias fields if you want to create a view. We should be able to infer this eventually but I have this implemented as such for now so that the types are really clear